.gitignore

```diff
@@ -110,3 +110,5 @@ ENV/
 #Eclipse project
 .project
 
+
+.DS_Store
```

Comment:
Is this change intentional ?

Reply:
It is @tomkralidis, despite just in favour of OSX users